tox.ini

```diff
@@ -39,7 +39,11 @@ commands =
 changedir = {toxinidir}/docs
 whitelist_externals = htmlproofer
                       gem
-commands =
+
+commands_pre =
     gem install html-proofer
+
+
+commands =
     sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
-    htmlproofer {envtmpdir}/html --alt-ignore '/.*/' --file-ignore '/2013/','/2014'/,'/2015/' --url-ignore '/video/' --allow-hash-href
+    htmlproofer {envtmpdir}/html/conf/australia/2018/about.html --alt-ignore '/.*/' --file-ignore "/2013/,/2014/,/2015/,/blog/" --url-swap ".html:/" --disable_external --url-ignore '/video/' --allow-hash-href
```

Comment:
DO we want the full path to `conf/australia/2018/about.html` here?

Reply:
That is just me testing on a single file, so I don't have to scroll through oodles of results.

Reply:
Will remove once I understand how to fix the problem.